src/views/tutorial-view/utils/variants/types.ts

```diff
@@ -1,12 +1,12 @@
-export interface TutorialVariant {
-	id: string
-	slug: string
-	name: string
-	options: TutorialVariantOption[]
-}
+import {
+	TutorialVariant as ClientTutorialVariant,
+	TutorialVariantOption as ClientTutorialVariantOption,
+} from 'lib/learn-client/types'
+
+// @TODO rename to TutorialViewVariant? Wait until rebase to avoid conflicts
 
-export interface TutorialVariantOption {
-	id: string
-	name: string
-	slug: string
+export interface TutorialVariant extends ClientTutorialVariant {
+	activeOption: TutorialVariantOption
 }
+
+export type TutorialVariantOption = ClientTutorialVariantOption
```

Comment:
I plan to adjust these type names so they dont duplicate the api client types in a follow up PR. It would just cause too much conflict with already open UI work to do so now.